### PR TITLE
Wormhole crates

### PR DESCRIPTION
**Pull request by hydrogenbond007**
- Added wormhole rust crates
- Updated pyth links

### CODE REVIEW

**Comment by hydrogenbond007** [comment]
@barnjamin bumping for review

File: scripts/src/chains/pythnet.json [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+  "homepage": "https://pyth.network/",
   "mainnet":{
     "name":"",
-    "id":"N/A"
+    "id":"26"
   },
   "testnet":{
     "name":"",
-    "id":"N/A"
+    "id":"[EVM testnet IDs](https://pyth.network/developers/price-feed-ids#pyth-evm-testnet)"
```
Comment (barnjamin) [inline scripts/src/chains/pythnet.json:10]: this is meant for the EIP155 chain id or in this case the genesis hash for solana platform chains

Comment (hydrogenbond007) [reply scripts/src/chains/pythnet.json:10]: Oh gotcha, So in this case do we put in hashes from the pythnet itself? becuase I don't think pyth has a public block explorer then how do we get the block hashes of different price feed smart contracts on pythnet? 
or we share this -> https://pyth.network/developers/price-feed-ids#solana-testnet

Comment (barnjamin) [reply scripts/src/chains/pythnet.json:10]: yea i don't think they do, we can leave this as N/A since I don't expect anyone will need to find this info

Comment (hydrogenbond007) [reply scripts/src/chains/pythnet.json]: gotcha, done

File: scripts/src/chains/pythnet.json [hunk 2]
```diff
@@ -1,14 +1,19 @@
 {
+  "title": "Pythnet",
+  "homepage": "https://pyth.network/",
   "mainnet":{
     "name":"",
-    "id":"N/A"
+    "id":"26"
```
Comment (barnjamin) [inline scripts/src/chains/pythnet.json:6]: This one needs to be reverted as well plz

Comment (hydrogenbond007) [reply scripts/src/chains/pythnet.json]: done, but I remember reading pyth id was 26 in the wormhole docs itself. Will try and find the link

File: docs/reference/environments/solana/README.md [hunk 3]
```diff
@@ -97,9 +99,9 @@ For more information see [https://docs.solana.com/cluster/commitments](https://d
 
 ## Ecosystem
 
-- No webpage, update [here](https://github.com/wormhole-foundation/docs.wormhole.com/blob/main/scripts/src/chains/pythnet.json)
-- No explorer, update [here](https://github.com/wormhole-foundation/docs.wormhole.com/blob/main/scripts/src/chains/pythnet.json)
-- No dev docs, update [here](https://github.com/wormhole-foundation/docs.wormhole.com/blob/main/scripts/src/chains/pythnet.json)
+- [Pythnet](https://pyth.network/), update [here](https://github.com/wormhole-foundation/docs.wormhole.com/blob/main/scripts/src/chains/pythnet.json)
+- Pyth Explorer, update [here](https://github.com/wormhole-foundation/docs.wormhole.com/blob/main/scripts/src/chains/pythnet.json)
```
Comment (barnjamin) [inline docs/reference/environments/solana/README.md:103]: Did you run the script to generate these? expecting it to say "No explorer"

Comment (hydrogenbond007) [reply docs/reference/environments/solana/README.md]: done

**Pull request merged**